docs/Contributing/API-for-contributors.md

```diff
@@ -668,19 +686,212 @@ Content-Type: application/octet-stream
 
 `Status: 204`
 
+### Update ABM token's teams
+
+`PATCH /api/v1/fleet/abm_tokens/:id/teams`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| id | integer | path | *Required* The ABM token's ID |
+| macos_team_id | integer | body | macOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+| ios_team_id | integer | body | iOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+| ipados_team_id | integer | body | iPadOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+
+#### Example
+
+`PATCH /api/v1/fleet/abm_tokens/1/teams`
+
+##### Request body
+
+```json
+{
+  "macos_team_id": 1,
+  "ios_team_id": 2,
+  "ipados_team_id": 3,
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+{}
+```
+
+### Renew ABM token
+
+`PATCH /api/v1/fleet/abm_tokens/:id/renew`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| id | integer | path | *Required* The ABM token's ID |
+
+#### Example
+
+`PATCH /api/v1/fleet/abm_tokens/1/renew`
+
+##### Request header
+
+```http
+Content-Length: 850
+Content-Type: multipart/form-data; boundary=------------------------f02md47480und42y
+```
 
-### Disable automatic enrollment (ADE)
+##### Request body
 
-`DELETE /api/v1/fleet/mdm/apple/abm_token`
+```http
+--------------------------f02md47480und42y
+Content-Disposition: form-data; name="token"; filename="server_token_abm.p7m"
+Content-Type: application/octet-stream
+
+<TOKEN_DATA>
+
+--------------------------f02md47480und42y
+```
+
+##### Default response
+
+`Status: 200`
```

Comment:
@marko-lisica should this response also include an empty object? The response for `PATCH /abm_tokens/:id/teams` [has an empty object](https://github.com/fleetdm/fleet/pull/21043/files#diff-831a63ebb3cab9b4e6b82d803d9ffcdc3722b12d486dcd7b13e9576643bb50b0R720).

Reply:
@jahzielv Is `PATCH /abm_tokens/:id/teams` supposed to always return empty object?

I guess not, it should return assigned teams after the latest change. Wdyt?

I think renew endpoint probably should return [same object](https://github.com/fleetdm/fleet/pull/21043/files/215cf3c78015183243418732081a5bf11bd0b541#diff-831a63ebb3cab9b4e6b82d803d9ffcdc3722b12d486dcd7b13e9576643bb50b0R663-R675) as Add ABM token endpoint

Reply:
That makes sense to me! As long as they're the same either way works for me.

Reply:
@jahzielv I updated following endpoints: 

- `POST /api/v1/fleet/abm_tokens`
- `PATCH /api/v1/fleet/abm_tokens/:id/renew`
- `PATCH /api/v1/fleet/abm_tokens/:id/teams`

They all now include JSON as the response (`abm_token` details after upload/update). I realized we do this in other endpoints, such as create/update policy or create/update query.

```json
"abm_token": {
  "id": 1,
  "apple_id": "apple@example.com",
  "org_name": "Fleet Device Management Inc.",
  "mdm_server_url": "https://example.com/mdm/apple/mdm",
  "renew_date": "2024-11-29T00:00:00Z",
  "terms_expired": false,
  "macos_team": "Workstations",
  "ios_team": "Company-owned iPhones",
  "ipados_team": "Company-owned iPads"
}
```

Reply:
Great thanks @marko-lisica !